src/ngraph/runtime/inference_engine/ie_tensor_view.hpp

```diff
@@ -0,0 +1,62 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <string>
+
+#include "ngraph/runtime/inference_engine/ie_backend_visibility.hpp"
+#include "ngraph/runtime/tensor.hpp"
+#include "ngraph/type/element_type.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace inference_engine
+        {
+            class IETensorView : public ngraph::runtime::Tensor
```

Comment:
the term TensorView is long dead. Please call this IETensor and rename the files to ie_tensor.?pp